lib/actions/deploy.coffee

```diff
@@ -6,6 +6,11 @@ getBuilderPushEndpoint = (baseUrl, owner, app) ->
 	escApp = encodeURIComponent(app)
 	"https://builder.#{baseUrl}/v1/push?owner=#{escOwner}&app=#{escApp}"
 
+getBuilderLogPushEndpoint = (baseUrl, buildId, owner, app) ->
+	escOwner = encodeURIComponent(owner)
+	escApp = encodeURIComponent(app)
+	"https://builder.#{baseUrl}/v1/pushLogs?owner=#{escOwner}&app=#{escApp}&buildId=#{buildId}"
```

Comment:
It might be worth using `querystring.stringify({ owner, app, buildId })` - https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options